internals/secrethub/service_gcp_link.go

```diff
@@ -53,6 +53,21 @@ func (cmd *ServiceGCPLinkCommand) Register(r command.Registerer) {
 	clause.Arg("namespace", "The SecretHub namespace to link.").Required().SetValue(&cmd.namespace)
 	clause.Arg("project-id", "The GCP project to link the namespace to.").Required().SetValue(&cmd.projectID)
 
+	clause.HelpLong("Linking a GCP project to a namespace is required to create SecretHub service accounts that use a GCP Service Account within the project. " +
+		"A SecretHub namespace can be linked to multiple GCP projects and a GCP project can be linked to multiple namespaces.\n" +
+		"\n" +
+		"As long as the link exists, new service accounts can be created for the GCP project. " +
+		"If a link is deleted, no new services can be created, but previously created services are unaffected. \n" +
+		"\n" +
+		"This command will open browser where you are asked to authorize SecretHub to perform iam.test on your GCP projects. " +
```

Comment:
```suggestion
		"This command will open your browser where you are asked to authorize SecretHub to perform iam.test on your GCP projects. " +
```

Alternatives: `a` or `the`